api/accounts.go

```diff
@@ -182,22 +182,22 @@ func (a *API) accountSetHandler(msg *apirest.APIdata, ctx *httprouter.HTTPContex
 	// Note that we enforce the metadata URI to be provided in the tx payload only if
 	// req.Metadata is provided, but not in the other direction.
 	if req.Metadata != nil && metadataURI == "" {
-		return fmt.Errorf("metadata provided but no metadata URI found in transaction")
+		return ErrNoMetadataURI
 	}
 
 	var metadataCID string
 	if req.Metadata != nil {
 		// if election metadata defined, check the format
 		metadata := AccountMetadata{}
 		if err := json.Unmarshal(req.Metadata, &metadata); err != nil {
-			return fmt.Errorf("wrong metadata format: %w", err)
+			return fmt.Errorf("%w: %w", ErrWrongMetadataFormat, err)
```

Comment:
I think we can ignore the warnings here; it's likely because our go vet is still go 1.19.

Reply:
seems we can't, go 1.19 fails to build, it's not just go vet acting out. so it seems we'll need go 1.20, until then i'll fallback to "%w %v"